tools/miaou.ml

```diff
@@ -440,8 +440,14 @@ module Make
       | Item txt ->
          Some
            (Item
-              (sprintf "\\expandafter\\MakeUppercase\\notthecase{%s}" txt))
-      | List _|DiffPair _|IfCond _ ->
+              (sprintf "\\expandafter{\\MakeUppercase\\notthecase{%s}}" txt))
+      | List (op,intro_txt,sep_txt,es) ->
+         (* eprintf "~~~~~~~~~~~~~~~~~~~~\n"; *)
+         (* eprintf "%s\n" (sprintf "\\expandafter{\\MakeUppercase\\notthecase{%s}}" intro_txt); *)
+         (* eprintf "%s\n" (List.fold_left (fun acc e -> acc ^ "," ^ (pp_txt e)) " " es) ; *)
+         (* eprintf "~~~~~~~~~~~~~~~~~~~~\n"; *)
```

Comment:
It looks like this was used for debugging can we remove it now? Or otherwise add that as code behind some debug flag?

Reply:
Not sure a debug flag is useful in this case (or rather, more concentrated effort is needed to understand which sort of debug info is helpful), these remnants were an oversight. Should be fixed now.